src/pages/api/bots/index.ts

```diff
@@ -36,6 +36,13 @@ export default async function handler(
           code: 401,
         });
       }
+      if (description.length < 200) {
+        return res.json({
+          error: "Description must be at least 200 characters long!",
+          status: "error",
+          code: 401,
```

Comment:
Sorry, missed this, change the code to `400` (Bad Request)

Reply:
fixed now